desktop.py

```diff
@@ -198,6 +198,12 @@ def make(opts: DesktopOpts, product: str, target_platform: str, target: str):
     if opts.strip_libs:
         strip_libs(opts, product, target_platform, target)
 
+def copy_bcl(opts: DesktopOpts, product: str, target_platform: str, target: str):
+    from distutils.dir_util import copy_tree
+    from bcl import get_profile_install_dirs
+    dest_dir = path_join(opts.install_dir, '%s-%s-%s' % (product, target, opts.configuration), 'lib/mono/4.5')
+    for src_dir in get_profile_install_dirs(opts, 'desktop'):
```

Comment:
We should raise a `BuildError` here if any of the profile source directories don't exist. And it would be great if it also said something like "Did you forget to build the BCL prior to this?" to make it obvious.